### ISSUE TITLE
Would we be better off automatically deploying the latest HEAD of `devel` to PyPi on Sunday nights?

### ISSUE THREAD

**Issue by ekalosak** [SheffieldML/GPy#1033]
People who install the bleeding edge will report breaking bugs that our tests don't cover. Low risk, can be disabled. The work required is no more than a cron GitHub Action with sufficient permissions to merge `devel` to `deploy`.

**Comment by lawrennd** [comment]
Interesting, I like the idea, are there nice examples of other projects doing something similar? Would be good to know their experience.

**Comment by MartinBubel** [comment]
Absolutely. I think releases should be triggered manually at this stage.
